rocketpy/prints/flight_prints.py

```diff
@@ -67,221 +81,171 @@ def initial_conditions(self):
 
     def out_of_rail_conditions(self):
-        """Prints out the Out of Rail Conditions available about the flight.
+        """Prints out the Out of Rail Conditions available about the flight,
+        including the time, velocity, stability margin, angle of attack, thrust
+        to weight ratio, and Reynolds number.
 
         Returns
         -------
         None
         """
-        if self.flight.post_processed is False:
-            self.flight.post_process()
         print("\nRail Departure State\n")
-        print("Rail Departure Time: {:.3f} s".format(self.flight.out_of_rail_time))
+        print(f"Rail Departure Time: {self.flight.out_of_rail_time:.3f} s")
+        print(f"Rail Departure Velocity: {self.flight.out_of_rail_velocity:.3f} m/s")
         print(
             "Rail Departure Stability Margin: "
             f"{self.flight.out_of_rail_stability_margin:.3f} c"
         )
         print(
-            "Rail Departure Stability Margin: {:.3f} c".format(
-                self.flight.stability_margin(self.flight.out_of_rail_time)
-            )
-        )
-        print(
-            "Rail Departure Angle of Attack: {:.3f}°".format(
-                self.flight.angle_of_attack(self.flight.out_of_rail_time)
-            )
+            "Rail Departure Angle of Attack: "
+            f"{self.flight.angle_of_attack(self.flight.out_of_rail_time):.3f}°"
         )
         print(
-            "Rail Departure Thrust-Weight Ratio: {:.3f}".format(
-                self.flight.rocket.thrust_to_weight(self.flight.out_of_rail_time)
-            )
+            "Rail Departure Thrust-Weight Ratio: "
+            f"{self.flight.rocket.thrust_to_weight(self.flight.out_of_rail_time):.3f}"
         )
         print(
-            "Rail Departure Reynolds Number: {:.3e}".format(
-                self.flight.reynolds_number(self.flight.out_of_rail_time)
-            )
+            "Rail Departure Reynolds Number: "
+            f"{self.flight.reynolds_number(self.flight.out_of_rail_time):.3e}"
         )
 
-        return None
-
     def burn_out_conditions(self):
-        """Prints out the Burn Out Conditions available about the flight.
+        """Prints out the Burn Out Conditions available about the flight,
+        including the burn out time, altitude, speed, freestream speed, Mach
+        number, and kinetic energy.
 
         Returns
         -------
         None
         """
         print("\nBurn out State\n")
-        print("Burn out time: {:.3f} s".format(self.flight.rocket.motor.burn_out_time))
+        print(f"Burn out time: {self.flight.rocket.motor.burn_out_time:.3f} s")
         print(
-            "Altitude at burn out: {:.3f} m (AGL)".format(
-                self.flight.z(self.flight.rocket.motor.burn_out_time)
-                - self.flight.env.elevation
-            )
+            "Altitude at burn out: "
+            f"{self.flight.altitude(self.flight.rocket.motor.burn_out_time):.3f} "
+            "m (AGL)"
```

Comment:
could add ASL / AGL information instead of just AGL

Reply:
[2a85ce8](https://github.com/RocketPy-Team/RocketPy/pull/579/commits/2a85ce8a40584d7a8f5cd95da12d3fed15fbc64f)